### DISCUSSION TITLE
CSV output

### DISCUSSION THREAD

**Discussion by NMuee** [prowler-cloud/prowler#2361] [Q&A] [answered]
Hello Prowler Team, 

Regarding Prowler 3.0 csv output
May I know in the future will there be enhancement to output the csv in a more standardize way? Similar to how Prowler 2.0 is done? 

I did a check on services (ecr) 
and the csv output i receive is a mixture of different column. 

It is kind of hard for me to filter the csv, and I don't use html as it is not convenient to make a report out of it. 

Thus, will there be any enhancement on this? 

Thank you

**Comment by jfagoagas** [comment]
Hi @NMuee, could you please attach the CSV output generated? What fields are mixed?

You can check the Prowler CSV output structure in the following link https://docs.prowler.cloud/en/latest/tutorials/reporting/#csv

Thanks for using Prowler 🚀

**Reply by NMuee** [reply]
[prowler-v3-ecr-output.csv](https://github.com/prowler-cloud/prowler/files/11484860/prowler-v3-ecr-output.csv)

I have uploaded an example output from ecr services scan. 
As you can see each checks has different number of column. 

Thank you for helping.

**Reply by jfagoagas** [reply]
Hi @NMuee, this is strange and I can't reproduce it in our environment.

What version of Prowler 3 are you using? Please use the latest one.

**Reply by NMuee** [reply]
I am pulling from the master branch with the latest commit
Not sure why my excel is this way. However, if i open the csv with my numbers app from Mac
It seems like the formatting is working as intended

**Reply by jfagoagas** [reply, marked as answer]
Now I understand, it is possible that when you are importing the CSV to Excel you are not setting the `;` as the delimiter and maybe some other options are not properly configured too. Please review your Excel configuration to import data coming from a CSV file.

I'm going to close this since the CSV format is being generated right.

Thanks for using Prowler 🚀
